lib/src/backend/platform_selector.dart

```diff
@@ -11,6 +11,7 @@ import 'test_platform.dart';
 /// The set of statically-known valid variable names.
 final _validVariables =
```

Comment:
I'm thinking we should change this variable name since we now pass a `validVariables` argument in validate. Maybe `_staticValidVariables`?

Reply:
Done.